registry/handler/handler.go

```diff
@@ -26,10 +27,15 @@ func (r *Registry) GetService(ctx context.Context, req *pb.GetRequest, rsp *pb.G
 }
 
 func (r *Registry) Register(ctx context.Context, req *pb.Service, rsp *pb.EmptyResponse) error {
-	err := r.Registry.Register(service.ToService(req))
+	var regOpts []registry.RegisterOption
+	ttl := time.Duration(req.Options.Ttl)
```

Comment:
This will need to be `time.Duration(req.Options.Ttl) x time.Second` otherwise it'll parse as nanoseconds.

Reply:
Truth!